### ISSUE TITLE
[Linux] Provide Debian/Ubuntu/PPA/Snap Repository

### ISSUE THREAD

**Issue by Philippe734** [UniversalMediaServer/UniversalMediaServer#1446]
Please, provide UMS in a official repository, as Debian or Ubuntu, or a Snap version, or at least through a PPA on Launchpad. After 5 years of development of UMS, it's time to jump to provide a classic install for Linux. In order to have an easy method to install of course, but also to keep up to date. Note for Snap, you can create a snap version directly from Github : https://build.snapcraft.io/

**Comment by Nadahar** [comment]
I don't think any of us knows enough about Linux packaging to be able to do that. There are also some potential issues in the fact that we bundle the transcoding engines, something I think several distros don't allow.

I somebody is willing to maintain a package, I'm sure we would do anything we could to help.

**Comment by valib** [comment]
I agree with Nadahar that we need somebody who will help us to do that.
